### PR TITLE
Rename hPA to hPa

### PR DESCRIPTION
**Pull request by dns2utf8**

### CODE REVIEW

**Comment by dbrgn** [comment]
Maybe we should keep a CHANGELOG file that would need to be updated with every change?

**Comment by dns2utf8** [comment]
Good point, will add one.

**Comment by dbrgn** [comment]
#9 is merged, feel free to rebase.

**Comment by dns2utf8** [comment]
Added and rebased

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,4 +9,4 @@ Changes should start with one of the following tags:
 
 ## v0.14 (unreleased)
 
-- ...
+- [changed] The unit `hPA` was renamed to `hPa` to match the SI unit
```
Comment (dbrgn) [inline CHANGELOG.md]: I'd mention where this unit is found:

> The unit `hPA` in `sensors.barometer.unit` was renamed to `hPa` to match the SI unit

**Comment by rnestler** [comment]
@dns2utf8 ping

**Pull request merged**